app/javascript/dashboard/routes/dashboard/contacts/components/ContactsTable.vue

```diff
@@ -80,7 +80,10 @@ export default {
                 status={row.availability_status}
               />
               <div>
-                <h6 class="sub-block-title user-name text-truncate">
+                <h6
+                  class="sub-block-title user-name text-truncate"
+                  style="width: 215px"
```

Comment:
We follow `rem` for hard coded width, please move to `style` and use `rem` instead of `px`